pyproject.toml

```diff
@@ -134,6 +134,7 @@ output-format="colorized"
 
 [tool.pylint.format]
 max-line-length="88"
+max-module-lines="1500"
```

Comment:
This seemed like a reasonable level — I'll be adding the same fix for privacy request verification in a subsequent PR. It's also likely worth moving the `ConsentRequest` code into its own module, for the sake of getting this back down to the default of 1000, but I'd rather do that afterwards as clean-up to get this fix out.